### DISCUSSION TITLE
We'd love to help maintain!

### DISCUSSION THREAD

**Discussion by seanspeaks** [serverless/serverless-google-cloudfunctions#303] [General]
Hey team... not sure who/how to get in touch, but we'd love to help maintain this project. "We" in this case is [Left Hook](https://lefthook.com), and we've built a framework ([The Frigg Framework](friggframework.org)) that leverages serverless to enable anyone to deploy integrations at scale. Since day one we've wanted the framework to be both serverless in architecture and cloud provider agnostic in nature, so that anyone can adopt it into their existing stack.

This plugin is key for that, for us.

Is that a @pgrzesik request/question? Who should we connect with?

**Comment by pgrzesik** [comment]
Hey @seanspeaks 👋 The best person to get in touch is probably @medikoo and/or @ac360. I was looking into modernizing this plugin, but unfortunately I was both too busy lately and it requires too much effort on my side. I'm happy to help in some capacity, but I'm in no way in charge of this plugin.

**Comment by medikoo** [comment]
@seanspeaks Great thanks for reaching out? Do you want to propose some new PR's with functionalities, help review the already submitted work?

We'll be happy to add you as maintainers, ideally if it starts with some PR proposals. We look forward!
